setupTests.js

```diff
@@ -68,5 +68,12 @@ window.z = {userPermission: {}};
 window.URL.createObjectURL = jest.fn();
 window.URL.revokeObjectURL = jest.fn();
 
+Object.defineProperty(document, 'elementFromPoint', {
```

Comment:
I fixed it globally for tests like `MessageList`, where we don't want to test this functionality, but it gets called now. It is still possible to mock it like `document.elementFromPoint = () => element;`